src/Arcus.WebApi.Security/Authentication/SharedAccessKeyAuthenticationAttribute.cs

```diff
@@ -1,13 +1,15 @@
 ﻿using System;
 using Arcus.Security.Secrets.Core.Interfaces;
 using GuardNet;
+using Microsoft.AspNetCore.Builder;
 using Microsoft.AspNetCore.Mvc;
 
 namespace Arcus.WebApi.Security.Authentication
 {
     /// <summary>
     /// Authentication filter to secure HTTP requests with shared access keys.
     /// </summary>
+    [Obsolete("Feature is moved to our 'Arcus.WebApi.Security.Authentication' NuGet package, please use Arcus.WebApi.Security.Authentication.SharedAccessKey.SharedAccessKeyAuthenticationAttribute")]
```

Comment:
👍 
I too always try to (1) describe the problem and (2) provide an alternative in obsolete functionality.
Cool!

Reply:
Thank you for that!

It's the least we can do 🤷‍♂️